src/colmap/estimators/bundle_adjustment.h

```diff
@@ -86,6 +86,19 @@ struct BundleAdjustmentOptions {
   int max_num_images_direct_dense_gpu_solver = 200;
   int max_num_images_direct_sparse_gpu_solver = 4000;
 
+  // Whether to use prior camera positions
```

Comment:
These options are specific to the `PosePriorBundleAdjuster`. Shouldn't they be part of `PosePriorBundleAdjuster::Options` instead? would this make propagating them from the CLI through the parent call stack more difficult?

Reply:
You were right, it is cleaner to have it directly into PosePriorBundleAdjuster::Options.  I have updated the sfm IncrementalMapper options to propagate those through the parent call stack.  Doing it like this feels right as many options are already propagated from the IncrementalPipelineOptions to the IncrementalMapper options.
Let me know what you think of it.